pkg/cli/admin/release/git.go

```diff
@@ -251,9 +252,21 @@ func mergeLogForRepo(g gitInterface, repo string, from, to string) ([]MergeCommi
 		args = []string{"log", "--no-merges", "--topo-order", "-z", "--pretty=format:%H %P%x1E%ct%x1E%s%x1E%b", fmt.Sprintf("%s..%s", from, to)}
 		out, err = g.exec(args...)
 		if err != nil {
-			return nil, gitOutputToError(err, out)
+			return nil, 0, gitOutputToError(err, out)
 		}
 		squash = true
+	} else {
+		// some repositories use both real merges and squash or rebase merging.
+		// Don't flood the output with single-commit noise, which might be poorly curated,
+		// but do at least mention the fact that there are non-merge commits in the
+		// first-parent line, so folks who are curious can click through to GitHub for
+		// details.
+		args := []string{"log", "--no-merges", "--first-parent", "-z", "--format=%H", fmt.Sprintf("%s..%s", from, to)}
```

Comment:
I assume that this filter out the commits that we will print and only select the ones that are discarded?